README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.com/Flaconi/docker-atlantis-terragrunt.svg?branch=master)](https://travis-ci.com/Flaconi/docker-atlantis-terragrunt)
+[![Build Status](https://github.com/Flaconi/docker-atlantis-terragrunt/workflows/Build-Publish/badge.svg)](https://github.com/Flaconi/docker-atlantis-terragrunt)
```

Comment:
```suggestion
[![Build Status](https://github.com/Flaconi/docker-atlantis-terragrunt/workflows/Build-Publish/badge.svg)](https://github.com/Flaconi/docker-atlantis-terragrunt/actions?query=workflow%3ABuild-Publish)
```

Reply:
This is to go directly to the actions on click

Reply:
Its a public repo, don't we want this?

Reply:
Actions are public as well anyway

Reply:
It's just the same behaviour we had with Travis bevor. Build badge link goes to buiild